index.bs

```diff
@@ -8632,7 +8632,7 @@ JavaScript code.
     1.  Let |p| be [=a new promise=] in |realm|.
     1.  Run the following steps [=in parallel=]:
         1.  Wait |ms| milliseconds.
-        1.  [=Resolve=] |p|.
+        1.  [=Queue a task=] on |taskSource| to [=resolve=] |p|.
```

Comment:
taskSource is not defined? Note that example 55 defines it, although in a kind of wishy-washy way.